lib/dependencies/ContextDependencyTemplateAsRequireCall.js

```diff
@@ -12,6 +12,11 @@ ContextDependencyTemplateAsRequireCall.prototype.apply = function(dep, source, o
 	var isAsync = dep.module && dep.module.async;
 	if(dep.module && (isAsync || containsDeps)) {
 		if(dep.valueRange) {
+			if(Array.isArray(dep.replaces))
+				for(var i = 0; i < dep.replaces.length; i++) {
+					var rep = dep.replaces[i];
+					source.replace(rep.range[0], rep.range[1] - 1, rep.value)
+				}
```

Comment:
This is a generalization of the `dep.prepend` that was needed to support template strings (used to replace the context part of the path with the in-context prefix).